WalletWasabi/WabiSabi/Client/WabiSabiHttpApiClient.cs

```diff
@@ -82,7 +86,19 @@ private async Task<HttpResponseMessage> SendWithRetriesAsync(RemoteAction action
 				using CancellationTokenSource requestCts = CancellationTokenSource.CreateLinkedTokenSource(combinedToken, requestTimeoutCts.Token);
 
 				// Any transport layer errors will throw an exception here.
-				HttpResponseMessage response = await _client.SendAsync(HttpMethod.Post, GetUriEndPoint(action), content, requestCts.Token).ConfigureAwait(false);
+				HttpResponseMessage response;
+				if (action != RemoteAction.GetHumanMonitor)
```

Comment:
It seems like a consensus arises on how to get the number of inputs. It should be added to the GetState request for every round. I suggest adding a simple number. Do you have a RegTest setup with Wasabi?

Reply:
We have to make sure to keep the compatibility with old clients.

Reply:
yes I was planning into making that as a first step PR this week-end